.evg.yml

```diff
@@ -76,7 +76,7 @@ tasks:
             echo "running client tests"
             cd stitch-js-sdk
             npm install
-            npm test
+            npm test -- --runInBand # force serial execution
```

Comment:
Note: Added this as a workaround for an occasional race condition that causes the mongodb service test to fail (Jest runs all test suites in parallel by default). I spent a bit of time trying to hunt down the root cause to no avail, but for the sake of unblocking this ticket + all the other V2 work, this bandaid should be sufficient, and I'll do some further test cleanup work separately which should take care of it. Running in serial only adds a negligible amount to the time taken to run the tests, since most of the suites complete on the order of a few ms.

Reply:
Sounds good. Maybe file a ticket to come back to this in case anyone is ever testing the SDK outside of MongoDB.